haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -13,16 +13,19 @@ have to click \"refresh\" or anything first)
 - BeShare supports any number of simultaneous uploads and downloads \
 in parallel, but will serialize simultaneous file requests to any \
 given host for efficiency."
+HOMEPAGE="https://github.com/HaikuArchives/BeShare"
 COPYRIGHT="1999-2012 Jeremy Friesner"
 LICENSE="
 	Public Domain
 	libsanta
 	"
-HOMEPAGE="https://github.com/HaikuArchives/BeShare"
-SOURCE_URI="$(HOMEPAGE)/archive/b721fcc.tar.gz"
+SOURCE_URI="${HOMEPAGE}/archive/b721fcc.tar.gz"
```

Comment:
I would certainly use srcGitRev, since several other recipes which point to a specific commit already use this very same (custom) parameter.
I see two advantages in using this: First, when we switch to a newer commit revision, the changes are in a single line. Secondly, the length of the _short_ alias might change in the future, so if/when such changes occur all recipes that use short commit refs would require an update.

Regarding the removal of catkeyseditor `0.1.0`, I think it would be better to keep it.
But I'm not against its removal if the project leaders say we should.